homeassistant/components/loqed/strings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Please enter a name for the lock and the internal Home Assistant URL.<BR/>Then visit https://app.loqed.com/API-Config and: <BR/>* Create an API-key: [add new api-key] <BR/>* Copy the [Integration information] value by clicking on the copy button in the detail-page of the just created API-key.",
```

Comment:
Please remove URLs from translations. If you want to use an URL in a translation, insert a placeholder and set that placeholder in the flow context. It makes it possible to change the URL later, without having to redo translations and also prevents someone from translating parts of the URL.